tests/constants/validators.js

```diff
@@ -4,7 +4,7 @@ const validator = index => ({
   selfStake: `10060000000${index}`,
   commission: 10000,
   validatorWeight: `100600000${index}000`,
-  address: 'lskhbxua8tpdckcewntcttfqfo4rbatampo2dgrno',
+  address: `lskhbxua8tpdckcewntcttfqfo4rbatampo2dgrno${index}`,
```

Comment:
I think there has be a better way of dynamically generating these addresses because the above would result in invalid addresses

Reply:
Unsure on how to do this and still making one endpoint work together another endpoint which needs to have same address. Best would be to have a test database of all our response types to make sure our mocks are in sync. But that would require a lot more code.